addresspoints/src/main/scala/grasshopper/addresspoints/AddressPointService.scala

```diff
@@ -1,6 +1,6 @@
 package grasshopper.addresspoints
 
-import addresspoints.api.Service
+import grasshopper.addresspoints.api.Service
```

Comment:
Shouldn't these packages all start with `gov.cfpb`?  We haven't built anything in Java/Scala at CFPB, so there's currently no standard.  Though,[qu](https://github.com/cfpb/qu) (written in Clojure) does take this same approach, starting package namespace with just `qu`.

For me, I always like to know where a package is coming from, and just `grasshopper` doesn't tell me much.  Also, I think `gov.cfpb.grasshopper` makes a better artifact `groupId` if/when we push this up to Maven Central or another repo at some point.  That said, there is currently no `grasshopper` project hosted there, so there won't be any conflicts if we do decide to stick with this as it is.